connection.go

```diff
@@ -67,8 +71,14 @@ func (c DatabaseConfig) Connection() (*Connection, error) {
 
 	addr := fmt.Sprintf("%s:%d", c.Host, c.Port)
 	connList := make([]*client.Conn, c.ConnectionMultiplier)
+	var options []func(*client.Conn)
+	if c.ClientMultiStatements {
+		options = append(options, func(c *client.Conn) {
+			c.SetCapability(mysql.CLIENT_MULTI_STATEMENTS)
```

Comment:
I was momentarily confused by the reuse of `c`.  I'd suggest either changing this one or -- maybe even better -- change the receiver parameter in this function and the previous one to something like `cfg` or `config` or whatever and leave this one as is.

Reply:
I agree that's a bad shadow.  I'll switch the receiver parameter to cfg.  The fact there's a DatabaseConfig and a Connection type in the file confused me on the "c" as well.